berkeley-function-call-leaderboard/bfcl/model_handler/actionGemma_handler.py

```diff
@@ -0,0 +1,144 @@
+import json
+
+from model_handler.oss_handler import OSSHandler
+from model_handler.model_style import ModelStyle
+
+SYSTEM_PROMPT ="""<bos>
+<start_of_turn>system
+You are an AI assistant for function calling. 
+For politically sensitive questions, security and privacy issues, 
+and other non-computer science questions, you will refuse to answer\n
+""".strip()
+
+TASK_INSTRUCTION = """
+You are an expert in composing functions. You are given a question and a set of possible functions. 
+Based on the question, you will need to make one or more function/tool calls to achieve the purpose. 
+If none of the function can be used, point it out and refuse to answer. 
+If the given question lacks the parameters required by the function, also point it out.
+""".strip()
+
+
+FORMAT_INSTRUCTION = """
+The output MUST strictly adhere to the following JSON format, and NO other text MUST be included.
+The example format is as follows. Please make sure the parameter type is correct. If no function call is needed, please make tool_calls an empty list '[]'
+```
+{
+  "tool_calls": [
+    {"name": "func_name1", "arguments": {"argument1": "value1", "argument2": "value2"}},
+    ... (more tool calls as required)
+  ]
+}
+```
+"""
+
+class ActionGemmaHandler(OSSHandler):
+    def __init__(self, model_name, temperature=0.001, top_p=1, max_tokens=512, dtype="bfloat16") -> None:
```

Comment:
According to the huggingface [model card](https://huggingface.co/KishoreK/ActionGemma-9B), the model is supposed to be run with `float32` instead of `bflaot16`.